packages/legend-graph/src/graph-manager/AbstractPureGraphManager.ts

```diff
@@ -156,6 +156,7 @@ export interface ExecutionOptions {
   serializationFormat?: EXECUTION_SERIALIZATION_FORMAT | undefined;
   parameterValues?: ParameterValue[];
   abortController?: AbortController | undefined;
+  preservedResponseHeaders?: string[];
```

Comment:
Nit: I think this should be
```suggestion
  preserveResponseHeaders?: string[];
```